### DISCUSSION TITLE
No window on startup

### DISCUSSION THREAD

**Discussion by torfason** [zed-industries/zed#6512] [Feature Requests]
When I start up, I'm getting no window. I can do file new, but if I start up the program, I always want a buffer to work on. 

I don't think this happened at first. I've tried going back to default settings but no luck.

**Comment by cha0ran** [comment]
Yes, I have encountered the same issue. I just enabled the vim mode and then disabled it later, and then Zed started without any window, and I couldn't fix it no matter how I modified the configuration.

**Comment by jansol** [comment]
See 
- #1148

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
